core/src/dird/dird.cc

```diff
@@ -174,21 +174,15 @@ int main(int argc, char* argv[])
   dir_app.add_flag("-m,--print-kaboom", prt_kaboom,
                    "Print kaboom output (for debugging).");
 
-  auto testconfig_option = dir_app.add_flag(
+  [[maybe_unused]] auto testconfig_option = dir_app.add_flag(
       "-t,--test-config", test_config, "Test - read configuration and exit.");
 
-#if !defined(HAVE_WIN32)
```

Comment:
the `#if` needs to stay; otherwise you will also get the option `--pid-file` on windows

Reply:
yes!